details.js

```diff
@@ -1,204 +1,763 @@
+	diagram[objs].push(scicos_link(scicos_link2));
+	diagram[objs].push(ifthel);
+	var scicos_link3={xx : new ScilabDouble(new data(351,0,0),new data(351,1,0),new data(450,2,0)), 
+					  yy : new ScilabDouble(new data(131.29,0,0),new data(93,1,0),new data(93,2,0)),
+					  ct : new ScilabDouble(new data(5,0,0),new data(-1,1,0)),
+					  from : new ScilabDouble(new data(7,0,0),new data(1,1,0)),
+					  to : new ScilabDouble(new data(3,0,0),new data(1,1,0))
+	};
+	diagram[objs].push(scicos_link(scicos_link3));
+	var scicos_link4={xx : new ScilabDouble(new data(262.57,0,0),new data(322.43,1,0)), 
+					  yy : new ScilabDouble(new data(163,0,0),new data(167,1,0)),
+					  ct : new ScilabDouble(new data(1,0,0),new data(1,1,0)),
+					  from : new ScilabDouble(new data(1,0,0),new data(1,1,0)),
+					  to : new ScilabDouble(new data(7,0,0),new data(1,1,0))
+	};
+	diagram[objs].push(scicos_link(scicos_link4));
+	diagram[objs].push(split);
+	var scicos_link5={xx : new ScilabDouble(new data(234,0,0),new data(234,1,0)), 
+					  yy : new ScilabDouble(new data(275.78,0,0),new data(198.71,1,0)),
+					  ct : new ScilabDouble(new data(5,0,0),new data(-1,1,0)),
+					  from : new ScilabDouble(new data(10,0,0),new data(1,1,0)),
+					  to : new ScilabDouble(new data(1,0,0),new data(2,1,0))
+	};
+	diagram[objs].push(scicos_link(scicos_link5));
+	var scicos_link6={xx : new ScilabDouble(new data(234,0,0),new data(361,1,0),new data(361,2,0)), 
+					  yy : new ScilabDouble(new data(275.78,0,0),new data(275.78,1,0),new data(202.71,2,0)),
+					  ct : new ScilabDouble(new data(5,0,0),new data(-1,1,0)),
+					  from : new ScilabDouble(new data(10,0,0),new data(2,1,0)),
+					  to : new ScilabDouble(new data(7,0,0),new data(1,1,0))
+	};
+	diagram[objs].push(scicos_link(scicos_link6));
+	
+	
+	var x=scicos_block();
+	x[gui]=new ScilabString(new data("ANDBLK",0,0));
+	x[graphics][sz]= new ScilabDouble(new data(2,0,0),new data(2,1,0));
+	x[graphics][gr_i]=new ScilabDouble();
+	x[graphics][pein]=new ScilabDouble(new data(0,0,0),new data(0,1,0));
+	x[graphics][peout]=new ScilabDouble(new data(0,0,0));
+	x[model][sim]= new ScilabString(new data("csuper",0,0));
+	x[model][evtin]=new ScilabDouble(new data(1,0,0), new data(1,1,0));
+	x[model][evtout]=new ScilabDouble(new data(1,0,0));
+	x[model][blocktype]= new ScilabString(new data("h",0,0));
+	x[model][firing]= new ScilabBoolean(new data("false",0,0));
+	x[model][dep_ut]= new ScilabBoolean(new data("false",0,0),new data("false",1,0));
+	x[model][rpar]=diagram;
+	return x;
+}
+
+function sci2exp(c)
+{
+	if(c.length==1)
+	return new ScilabString(new data(c.toString(),0,0));
+	else
+	return new ScilabString(new data("["+c.toString()+"]",0,0));
+}
+
+
+function C()
+{
+	var i=0;
+	var arr=[]
+	for(i=0;i<arguments[0].length;i++)
+	{
+		arr.push(new data(arguments[0][i],i,0));
+	}
+	return new ScilabDouble(...arr);
+}
+
+function CONST_m()
+{
+	var c=[1];
+	switch(arguments[0])
+	{
+		
+		case "get":
+			var array=["Constant_Value"];
```

Comment:
Send id's to the User Interface - This can be found in the return parameters of scicos_getvalue. 

These will be passed to the user within the input tags and subsequently be retrieved.